vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -41,6 +41,10 @@ def textual_dns_policy
     {:label => "DNS Policy", :value => @record.dns_policy}
   end
 
+  def textual_ip
+    {:label => "IP", :value => @record.ip}
```

Comment:
@zeari , please name it "IP Address"  in UI to make it consistent with Host and VM screens that have a similar field.